core/frontend/src/libs/notifier.ts

```diff
@@ -0,0 +1,44 @@
+import notifications from '@/store/notifications'
```

Comment:
I liked the simplification in use. Fewer lines to do the same thing.
Didn't get why this is separated from the store, though. Seems to me, they have the same scope and this is just a helper, so would be better in the same file.

Reply:
there is a rule on the project that does not allow multiple classes definitions on the same file.
https://eslint.org/docs/rules/max-classes-per-file